domains/gingdev.json

```diff
@@ -7,7 +7,7 @@
   },
   "record": {
     "A": [
-      "204.246.56.80"
+      "192.243.108.102"
```

Comment:
`192.xxx.xxx.xxx` is not a valid public A record. This won't be accessible outside your network